### ISSUE TITLE
one problem

### ISSUE THREAD

**Issue by birdsheng** [MCG-NJU/TDN#20]
hi, if you load a model (--resume) which epoch is before warm epoch,  your lr scheduler within the warm epoch is right, but when it enters the after scheduler, the lr update is wrong , the entry in the lr_scheduler.py of line 29 can not enter. Can you fix it?? thank you.

**Comment by yztongzhan** [comment]
Hi @birdsheng Thanks for your suggestion. 
1. I think this specific problem occur due to load lr_scheduler `scheduler.load_state_dict(checkpoint['scheduler'])`
2. we didn't use `warmup epoch` when train our TDN.

**Comment by birdsheng** [comment]
ok, that is solved
